plugins/inputs/gnmi/gnmi.go

```diff
@@ -7,6 +7,7 @@ import (
 	_ "embed"
 	"errors"
 	"fmt"
+	"github.com/influxdata/telegraf/plugins/inputs"
```

Comment:
Could this import be moved to the telegraf imports section?

Reply:
Yes, I've missed that.